test/new_tests/test_admin_set_quotas.py

```diff
@@ -29,8 +27,7 @@ def setup_method(self, method):
         try:
             self.client.admin_create_role("usr-sys-admin-test", usr_sys_admin_privs, write_quota=4500)
         except e.QuotasNotEnabled:
-            pytest.mark.skip(reason="Got QuotasNotEnabled, skipping quota test.")
-            pytest.skip()
+            pytest.skip(reason="Got QuotasNotEnabled, skipping quota test.")
```

Comment:
You changed the test code from calling both pytest.mark.skip() and pytest.skip() to just pytest.skip().  I think having both is redundant, so good to remove one, but isn't pytest.mark.skip() better?

Reply:
Hey Justin, pytest.mark.skip() is supposed to be put above the function. It's not supposed to be used inside the function.
Docs: https://docs.pytest.org/en/latest/how-to/skipping.html#skipping-test-functions